script/Core/Util/Guid.lua

```diff
@@ -0,0 +1,11 @@
+function Guid()
```

Comment:
We should discuss if we wanna have a UUID Guid system or just use u64 (like the one in libphx right now)